histomicstk/segmentation/nuclear/detect_nuclei_kofahi.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import scipy as sp
+import skimage.morphology
+import histomicstk.filters.shape as htk_shape_filters
+import histomicstk as htk
+
+
+def detect_nuclei_kofahi(im_nuclei_stain, foreground_threshold, min_radius,
```

Comment:
change the name of this function to `kofahi_alog_lmc` to summarize the key idea of the method in the func name.